### ISSUE TITLE
Indicate the expected field patterns to user

### ISSUE THREAD

**Issue by marienfressinaud** [FreshRSS/FreshRSS#773]
Users must be aware of what forms expect from us (e.g. at least 7 characters for a password field).

See https://github.com/FreshRSS/FreshRSS/issues/769 for an example.

**Comment by marienfressinaud** [comment]
By the way, I have created a new label for easy tickets.

**Comment by aledeg** [comment]
Where should it happen? If it's on the user management page, it looks like it is already done!
![user](https://cloud.githubusercontent.com/assets/3056148/6317174/f8fb3450-ba13-11e4-9b83-6882181f1340.png)

If it's not, could you explicit a little bit more?

**Comment by marienfressinaud** [comment]
Well, I don't remember exactly why I opened this ticket after #769 ^^'. But it seems to me that we should add information on database configuration page during installation, it is the last page which don't include enough information about forms.
